frontend/src/Components/Register.jsx

```diff
@@ -0,0 +1,151 @@
+              className="p-2 border rounded w-full outline-none"
+              autoComplete="off"
+            />
+          </div>
+
+          {/* Confirm Password */}
+          <div className="mb-4">
+            <label htmlFor="confirmPassword" className="block text-white mb-1">
+              Confirm Password
+            </label>
+            <input
+              type="password"
+              id="confirmPassword"
+              name="confirmPassword"
+              className="p-2 border rounded w-full outline-none"
+              autoComplete="off"
+            />
+          </div>
+
+          {/* Address */}
+          <div className="mb-4">
+            <label htmlFor="address" className="block text-white mb-1">
+              Address
+            </label>
+            <input
+              type="text"
+              id="address"
+              name="address"
+              className="p-2 border rounded w-full outline-none"
+              autoComplete="off"
+            />
+          </div>
+
+          {/* Phone Number */}
+          <div className="mb-4">
+            <label htmlFor="phoneNumber" className="block text-white mb-1">
+              Phone Number
+            </label>
+            <input
+              type="tel"
+              id="phoneNumber"
+              name="phoneNumber"
+              className="p-2 border rounded w-full outline-none"
+              autoComplete="off"
+            />
+          </div>
+
+          {/* Number of Employees */}
+          <div className="mb-4">
+            <label htmlFor="numEmployees" className="block text-white mb-1">
+              No of Employees
+            </label>
+            <select
+              id="numEmployees"
+              name="numEmployees"
+              className="p-2 border rounded w-full outline-none"
+              autoComplete="off"
+            >
+              {Array.from({ length: 30 }, (_, i) => (
+                <option key={i + 1} value={i + 1}>
+                  {i + 1}
+                </option>
+              ))}
+            </select>
+          </div>
+
+          {/* Number of HRs */}
+          <div className="mb-4">
+            <label htmlFor="numHRs" className="block text-white mb-1">
+              No of HRs
+            </label>
+            <select id="numHRs" name="numHRs" className="p-2 border rounded w-full outline-none" autoComplete="off">
+              {Array.from({ length: 10 }, (_, i) => (
+                <option key={i + 1} value={i + 1}>
+                  {i + 1}
+                </option>
+              ))}
+            </select>
+          </div>
+
+          {/* Submit Button */}
+          <button
+            type="submit"
+            className="flex justify-center sm:col-span-2 bg-sec-color text-white px-4 py-2 rounded cursor-pointer mt-6"
+          >
+            <ExitToAppOutlinedIcon className="mr-1"></ExitToAppOutlinedIcon>
+            <p className="text-1xl font-bold">Register</p>
```

Comment:
Please check again this button positioning. It's looking out of the right portion and the screen is scrollable. Also, I think the text fields have extra height.